interop-tests/src/bin/ping.rs

```diff
@@ -168,17 +168,14 @@ fn secure_channel_protocol_from_env<C: AsyncRead + AsyncWrite + Unpin + Send + '
     identity: &identity::Keypair,
 ) -> Result<
     MapOutboundUpgrade<
-        MapInboundUpgrade<
-            Either<noise::NoiseAuthenticated<XX, X25519Spec, ()>, tls::Config>,
-            MapSecOutputFn<C>,
-        >,
+        MapInboundUpgrade<Either<noise::Config, tls::Config>, MapSecOutputFn<C>>,
         MapSecOutputFn<C>,
     >,
 > {
     let either_sec_upgrade = match from_env("security")? {
-        SecProtocol::Noise => Either::Left(
-            noise::NoiseAuthenticated::xx(identity).context("failed to intialise noise")?,
-        ),
+        SecProtocol::Noise => {
+            Either::Left(noise::Config::new(identity).context("failed to intialise noise")?)
```

Comment:
Same here, needs a bump of the `libp2p-noise` dependency.

Reply:
Done through bumping the noise version in `libp2p`.

Reply:
I am confused, would you then not have to bump the `libp2p` version in `interop-tests` `Cargo.toml`?

Reply:
You are absolutely correct. My bad!

Reply:
Thanks for checking :pray:. Resolving here.